src/gui/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
-cmake_minimum_required (VERSION 3.4)
```

Comment:
This should be in a separate PR.

Reply:
ok

Reply:
Split in to https://github.com/input-leap/input-leap/pull/1714